session_test.go

```diff
@@ -70,7 +70,7 @@ func TestRequestPriority(t *testing.T) {
 			}
 
 			// Read-Write Transaction.
-			if err := session.BeginReadWriteTransaction(test.transactionPriority); err != nil {
+			if err := session.BeginReadWriteTransaction(test.transactionPriority, "app=spanner-cli,env=test"); err != nil {
```

Comment:
Can we simply put an empty string `""` to the tag here? We want to keep this test focusing on the transaction priority.

Reply:
fixed.